amida-tech/indaba-client
# Feature/inba 877 grommet list components

If this PR fixes a bug, you _must_ add test cases representative of the bug.

Please refer to [Tettra](https://app.tettra.co/teams/amida/pages/amida-pull-request-and-code-review-guide) for PR review guidelines.

#### What does this PR do?
- Replaces the use of List and ListItem via grommet in the FilteredList location
- Refactors the grommet implementation with a pure JSX implementation in the Workflow/UserSidebar location

#### Related JIRA tickets:
- https://jira.amida-tech.com/browse/INBA-877

#### How should this be manually tested?
- Log in as PM:
- Go into a Project and assign it a couple groups (i.e. QA Testers, Commenters, and Reviewers)
- Add users to the groups using the side-by-side selector and ensure everything works properly
- In the Project's Workflow view, ensure the list of users shows up in the sidebar and that everything filters correctly when selecting groups or searching for individual users

#### Background/Context
- Note that for the List, multiple select functionality has been removed and will need to be re-implemented somehow (shouldn't be that hard) if simply tracking `events`

#### Screenshots (if appropriate):
